main/guides/orchestration/index.md

```diff
@@ -1,11 +1,20 @@
+# What is Agoric Orchestration?
+
+Agoric’s Orchestration capability allows developers to easily build cross-chain capabilities into existing applications or to create novel cross-chain-focused products.
+
+The Agoric Orchestration API sits on top of Agoric’s novel VM that provides three key elements that make multichain applications possible:
+
+- **Remote account control and transfer**: Use the Orchestration APIs to easily create accounts on remote chains, transfer assets, and invoke contracts. Your Agoric contract orchestrates all behavior directly.
```

Comment:
```suggestion
- **Remote account control and transfer**: Use the Orchestration API to easily create accounts on remote chains, transfer assets, and invoke contracts. Your Agoric contract orchestrates all behavior directly.
```

Reply:
thoughts here, I think we are speaking about the singular use of API here, the orchestration api as a single concept, not the individual apis